src/components/event-listing/EventListing.tsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Card, Divider, Button } from 'antd';
+
+// placeholder interface for now (will replace with real interface once hooked up to db)
+interface Event {
+  title: string;
+  date: string;
+  time: string;
+  description: string;
+  otherNotes?: string;
+  // TODO: use correct file type
+  thumbnail?: string;
+}
+
+const StyledCard = styled(Card)`
+  margin-bottom: 12px;
+`; 
+
+const CardContent = styled.div`
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+`;
+
+const H1 = styled.h1`
+  font-weight: bold;
+`;
+
+const H3 = styled.h1`
+  margin-bottom: 2px;
+`;
+
+const ThinDivider = styled(Divider)`
+  margin: 5px;
+  width: 100%;
+`;
+
+const GreenButton = styled(Button)`
+  color: white;
+  background-color: #2d870d;
+`;
+
+const Thumbnail = styled.img`
+  margin-right: 20px;
+  width: 20%;
+`;
+
+const Info = styled.div`
+  width: 80%;
+`;
+
+const EventListing: React.FC<{event: Event}> = ( props ) => {
```

Comment:
```suggestion
const EventListing: React.FC<Event> = ( { title, description, thumbnail  ) => {
```
Two suggestions: 
1. You can just make the type of the component the interface itself, there's no need to specify the event as a field.  Typically, a naming convention is `const FooBar: React.FC<FooBarProps>`, so I'd suggest renaming the interface to `EventListingProps`. 
2. You can destructure the props inline, so that in the component you can reference `title`, `thumbnail`, etc directly.